### ISSUE TITLE
pr feature name is chinese failed 404

### ISSUE THREAD

**Issue by M2shad0w** [go-gitea/gitea#3303]
./gitea -v
Gitea version 08cf7d9 built with: bindata, sqlite

## Description

ERROR LOG
```
[Macaron] 2018-01-05 17:42:54: Started POST $NAME/compare/develop...feature/%25e4%25b8%25ad%25e6%2596%2587%25e7%2589%25b9%25e6%2580%25a7 for 172.16.10.118
[Macaron] 2018-01-05 17:42:54: Completed POST $NAME/compare/develop...feature/%25e4%25b8%25ad%25e6%2596%2587%25e7%2589%25b9%25e6%2580%25a7 404 Not Found in 33.403759ms

```

**Comment by M2shad0w** [comment]
templates/repo/issue/new_form.tmpl

**Comment by M2shad0w** [comment]
![error_path](https://user-images.githubusercontent.com/3387446/34607707-c003340a-f24f-11e7-8f94-9824edf38aeb.png)

**Comment by M2shad0w** [comment]
error path should be `%e4%b8%ad%e6%96%87%e7%89%b9%e6%80%a7` not `%25e4%25b8%25ad%25e6%2596%2587%25e7%2589%25b9%25e6%2580%25a7`

**Comment by M2shad0w** [comment]
i used the gogs templates and upgrade by gogs version
maybe the error is caused by 
`<form class="ui comment form stackable grid" action="{{EscapePound .Link}}" method="post">`
and the function
https://github.com/go-gitea/gitea/blob/f2e20c81b66e6a937ecdb686f8d1011371433365/modules/templates/helper.go#L112
  
```
	str := "...feature/%25e4%25b8%25ad%25e6%2596%2587%25e7%2589%25b9%25e6%2580%25a7"
	c := strings.NewReplacer("%25", "%", "#", "%23", " ", "%20").Replace(str)
```

**Comment by lafriks** [comment]
Fixed by #3311
